pom.xml

```diff
@@ -129,6 +130,11 @@
                 <artifactId>quarkiverse-apicurio-registry-client</artifactId>
                 <version>${quarkiverse.apicurio.registry.client.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.confluent</groupId>
+                <artifactId>kafka-avro-serializer</artifactId>
+                <version>${confluent.kafka-avro-serializer.version}</version>
+            </dependency>
```

Comment:
The dependency is here, but the repository where these dependencies are in is declared in a submodule. This is wrong as it would require other submodules to register the repository every time a submodule needs a confluent dependency.